app/src/components/tenantsRightsModal.js

```diff
@@ -10,6 +10,11 @@ Handlebars.registerPartial("trGroups", template);
 export const ERROR_MISSING_COORDS =
   "Missing coordinates from address search result";
 
+/** Handles
+ * - rendering the tenants rights group content in the corresponding modal dialog instance
+ * - toggling whether slide 8 says there are tenants rights groups in the search result area
+ * NOTE: modal dialog functionality handled by ModalDialog component
+ */
 export class TenantsRightsModal extends Component {
```

Comment:
TODO: rename TenantsRightsModal (and related hbs & scss partials) to something more descriptive such as "tenants rights groups" or "slide 8 get help"?